starknet_py/tests/e2e/docs/guide/test_multicall.py

```diff
@@ -0,0 +1,43 @@
+import pytest
+
+
+@pytest.mark.asyncio
+async def test_multicall(account, deployed_balance_contract):
+    # pylint: disable=import-outside-toplevel
+    balance_contract = deployed_balance_contract
+    (initial_balance,) = await balance_contract.functions["get_balance"].call()
+    # docs: start
+    from starknet_py.hash.selector import get_selector_from_name
+    from starknet_py.net.client_models import Call
+
+    calls = [
+        balance_contract.functions["increase_balance"].prepare(amount=10),
+        balance_contract.functions["increase_balance"].prepare(amount=30),
+        balance_contract.functions["increase_balance"].prepare(amount=50),
+    ]
```

Comment:
I think it'd make more sense to have a new subsection in "Using existing contracts" where we can show that prepared calls from Contract can be used in BaseAccount.